lib/tentacat.ex

```diff
@@ -101,8 +103,8 @@ defmodule Tentacat do
     Application.get_env(:tentacat, :request_options, [])
   end
 
-  defp extra_headers do
-    Application.get_env(:tentacat, :extra_headers, [])
+  defp extra_headers(headers) do
+    Application.get_env(:tentacat, :extra_headers, headers)
```

Comment:
I was wondering if here we would like to concatenate `extra_headers` with `headers`?